packages/server/postgres/migrations/1709860378296_embeddingsMetadataId.ts

```diff
@@ -0,0 +1,41 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    ALTER TABLE "EmbeddingsMetadata"
+      DROP COLUMN "models",
+      ALTER COLUMN "refId" SET NOT NULL;
```

Comment:
@jordanh just to confirm-- was there any reason this was nullable?

Reply:
No, you're right to alter this to NOT NULL